wrappers/services.go

```diff
@@ -1880,21 +1939,38 @@ func RestartServices(svcs []*snap.AppInfo, explicitServices []string,
 	return nil
 }
 
+// ServicesEnableState returns a map of service names from the given snap,
+// together with their enable/disable status.
+// XXX: Not actually used by any code AFAICT. Remove this.
```

Comment:
isn't it used in overlord/snapstate/backend/link.go ?

Reply:
It's called by `Backend.ServicesEnableState()` - But it does not look like anything calls the backend function? I may have missed this, but I cannot find any uses.

Reply:
Moved the XXX to the backend function instead